### ISSUE TITLE
Unexpected CFBundleExecutable Key

### ISSUE THREAD

**Issue by yassipad** [NativeScript/NativeScript#7670]
**Environment**
✔ Component nativescript has 6.0.3 version and is up to date.
✔ Component tns-core-modules has 6.0.6 version and is up to date.
✔ Component tns-android has 6.0.1 version and is up to date.
✔ Component tns-ios has 6.0.2 version and is up to date.

 - Plugin(s):
"nativescript-app-sync": "^1.0.4"
"nativescript-calendar": "^2.1.0"
"nativescript-directions": "^1.3.1"
"nativescript-fingerprint-auth": "^7.0.1"
"nativescript-gif": "^4.0.2"
"nativescript-image-cache-it": "^5.0.0-alpha.3"
"nativescript-image-swipe": "^5.0.0"
"nativescript-iqkeyboardmanager": "^1.5.1"
"nativescript-mapbox": "^4.4.1"
"nativescript-secure-storage": "^2.5.0"
"nativescript-social-login": "^4.0.1"
"nativescript-social-share": "^1.5.2"
"nativescript-stripe-sdk": "^1.1.4"
"nativescript-videoplayer": "^4.2.1"
"nativescript-vue": "^2.4.0"

**Describe the bug**
After migrating to Nativescript 6, I am unable to publish my iOS app. I have tested publishing it through XCode and everything works fine except I eventually get the following error: ITMS-90535: Unexpected CFBundleExecutable Key - The bundle at '/Payload/frontend.app/app/App_Resources/iOS/Info.plist' does not contain a bundle executable.

Using "tns publish ios", I get the following (very strange) error: {"data":null,"messages":{"warn":null,"error":["Unexpected Error"],"info":null},"statusCode":"ERROR"}

What I find strange is that I am perfectly able to run the app both on emulator and on my iPhone (btw great work on ns6, hmr works like a charm ❤️), I simply cannot publish it...

**Comment by rigor789** [comment]
I ran into this exact issue very recently. For me the issue was that the `App_Resources` folder was being copied to the output folder.

I commented out the following section of my `webpack.config.js`
```js
            // Copy native app resources to out dir.
            new CopyWebpackPlugin([{
                from: `${appResourcesFullPath}/${appResourcesPlatformDir}`,
                to: `${dist}/App_Resources/${appResourcesPlatformDir}`,
                context: projectRoot,
            }]),
```

With this commented out, the app worked fine, and is now on TestFlight without issues.

I also moved the `App_Resources` folder out of the `app` folder, and tweaked the `nsconfig.json` to accommodate this change, but I don't think that affected the build outcome.

**Comment by yassipad** [comment]
Yes, thank you! I'm still getting the same error using "tns publish ios" but i can successfully submit my app using XCode. However, I now have another issue, which is that the app crashes right after the launchscreen...

**Comment by yassipad** [comment]
Ok, for anyone getting that error, I think I found the reason: I'm also using **vue-cli-plugin-nativescript-vue** and since NS6 brings HMR by default, I had to use tns publish with hmr option set to false:

```
tns publish ios --hmr=false
```

Everything works fine now :)

**Comment by rigor789** [comment]
I suggest using `--release` as that will minify the code, and I believe also disable hmr.
